apps/okreads/browser/src/app/app.component.html

```diff
@@ -5,6 +5,7 @@
         <h1>okreads</h1>
         <div>
           <button
+            aria-label="reading list"
```

Comment:
Are you sure this button actually needs an `aria-label`?

(Hint: it doesn't. Why not?)

Reply:
What about the button on line 27 below?

Reply:
Removed: will override the visible text.